api.planx.uk/modules/auth/controller.ts

```diff
@@ -70,3 +73,30 @@ function setJWTSearchParams(returnTo: string, res: Response, req: Request) {
   url.searchParams.set("jwt", req.user!.jwt);
   res.redirect(url.href);
 }
+
+/**
+ * Revokes a user's JWT on logout
+ * TODO: We check if a JWT is revoked when authenticating requests via the API and Hasura
+ */
+export const logout: RequestHandler = async (_req, res, next) => {
+  const { jwt, sub: userId } = userContext.getStore()?.user || {};
+
+  if (!jwt) {
+    return next(
+      new ServerError({
+        message: `JWT missing from logout request, no token to revoke for user ${userId}`,
+      }),
+    );
+  }
```

Comment:
I'm pretty sure this is unreachable, but keeping it here for peace of mind, being explicit with the intentions of the code, and for type narrowing.

The `useLoginAuth()` middleware called as part of the route will reject if there's no JWT - tests prove this behaviour.